$NetBSD: patch-configure.ac,v 1.1 2023/06/17 11:25:19 riastradh Exp $

Get native compiler for executable compiled and run at build-time.

Add missing configure check for rl_echo_signal_char.

--- configure.ac.orig	2023-04-18 19:55:55.000000000 +0000
+++ configure.ac
@@ -17,6 +17,7 @@ AM_INIT_AUTOMAKE([foreign])
 AC_SUBST([RELEASE_DATE])
 dnl Checks for programs.
 AC_PROG_CC
+AX_PROG_CC_FOR_BUILD
 AC_PROG_INSTALL
 AC_PROG_LN_S
 AM_PROG_LEX
@@ -117,6 +118,8 @@ if test "x$ac_cv_header_readline_readlin
 		  AC_DEFINE(HAVE_RL_RESET_AFTER_SIGNAL,1,[Has rl_reset_after_signal]),,[#include <readline/readline.h>])
     AC_CHECK_DECL(rl_cleanup_after_signal,
 		  AC_DEFINE(HAVE_RL_CLEANUP_AFTER_SIGNAL,1,[Has rl_cleanup_after_signal]),,[#include <readline/readline.h>])
+    AC_CHECK_DECL(rl_echo_signal_char,
+		  AC_DEFINE(HAVE_RL_ECHO_SIGNAL_CHAR,1,[Has rl_echo_signal_char]),,[#include <readline/readline.h>])
 fi
 
 if test "x$prefix" = xNONE; then
